.generator/src/generator/templates/api.j2

```diff
@@ -57,47 +49,12 @@ func (r *{{ operation.operationId }}OptionalParameters) With{{ name|camel_case }
 }
 {%- endfor %}
 
-func (a *{{ classname }}) build{{ operation.operationId }}Request(ctx _context.Context{% for name, parameter in operation|parameters if parameter.required or parameter.in == "path" %}, {{ name|variable_name}} {{ get_type_for_parameter(parameter) }}{% endfor %}{%- for name, parameter in operation|parameters if not parameter.required and parameter.in != "path" %}{%- if loop.first %}, o ...{{ operation.operationId }}OptionalParameters{% endif %}{% endfor %}) (api{{ operation.operationId }}Request, error) {
-	req := api{{ operation.operationId }}Request{
-		ctx:        ctx,
-		{%- for name, parameter in operation|parameters if parameter.required or parameter.in == "path" %}
-		{%- if loop.first %}
-		
-		{%- endif %}
-		{{ name|variable_name}}: {% if parameter.in != "path" %}&{% endif %}{{ name|variable_name}},
-		{%- endfor %}
-	}
-	{%- for name, parameter in operation|parameters if not parameter.required and parameter.in != "path" %}
-	{%- if loop.first %}
-
-	if len(o) > 1 {
-		return req, {{ common_package_name }}.ReportError("only one argument of type {{ operation.operationId }}OptionalParameters is allowed")
-	}
-
-	if o != nil {
-	{%- endif %}
-		req.{{ name|variable_name}} = o[0].{{ name|attribute_name }}
-	{%- if loop.last %}
-	}
-	{%- endif %}
-	{%- endfor %}
-	return req, nil
-}
-
 // {{ operation.operationId }} {{ operation.summary }}.
 {{ operation.description|block_comment(prefix="//") }}{% if operation.deprecated %}
 //
 // Deprecated: This API is deprecated.{% endif %}
 func (a *{{ classname }}) {{ operation.operationId }}(ctx _context.Context{% for name, parameter in operation|parameters if parameter.required or parameter.in == "path" %}, {{ name|variable_name}} {{ get_type_for_parameter(parameter) }}{% endfor %}{%- for name, parameter in operation|parameters if not parameter.required and parameter.in != "path" %}{%- if loop.first %}, o ...{{ operation.operationId }}OptionalParameters{% endif %}{% endfor %}) ({% if returnType %}{{ returnType }}, {% endif %}*_nethttp.Response, error) {
-	req, err := a.build{{ operation.operationId }}Request(ctx{% for name, parameter in operation|parameters if parameter.required or parameter.in == "path" %}, {{ name|variable_name}}{% endfor %}{%- for name, parameter in operation|parameters if not parameter.required and parameter.in != "path" %}{%- if loop.first %}, o...{% endif %}{% endfor %})
-	if err != nil {
-		{%- if returnType %}
-		var localVarReturnValue {{ returnType }}
-		{%- endif %}
-		return {% if returnType %}localVarReturnValue, {% endif %}nil, err
-	}
-
-	return a.{{ operation.operationId|untitle_case }}Execute(req)
+	return a.{{ operation.operationId|untitle_case }}Execute(ctx{% for name, parameter in operation|parameters if parameter.required or parameter.in == "path" %}, {{ name|variable_name}}{% endfor %}{%- for name, parameter in operation|parameters if not parameter.required and parameter.in != "path" %}{%- if loop.first %}, o...{% endif %}{% endfor %})
```

Comment:
I think we can get rid of the execute method then? We just need to remove the reference in the pagination implantation I think.